src/System.Private.CoreLib/shared/System/AppDomain.cs

```diff
@@ -289,7 +290,7 @@ public ObjectHandle CreateInstance(string assemblyName, string typeName)
             return Activator.CreateInstance(assemblyName, typeName);
         }
 
-        public ObjectHandle CreateInstance(string assemblyName, string typeName, bool ignoreCase, BindingFlags bindingAttr, Binder binder, object[] args, System.Globalization.CultureInfo culture, object[] activationAttributes)
+        public ObjectHandle CreateInstance(string assemblyName, string typeName, bool ignoreCase, BindingFlags bindingAttr, Binder? binder, object?[]? args, System.Globalization.CultureInfo? culture, object?[]? activationAttributes)
```

Comment:
All these `CreateInstance` apis could return null right?

https://source.dot.net/#System.Private.CoreLib/shared/System/Activator.RuntimeType.cs,140

Reply:
> All these CreateInstance apis could return null right?

I didn't think so; the docs don't make any mention of that, and tracing it through I don't see how they would, but it's possible I missed a path.  @jkotas, are there circumstances in which Activator.CreateInstance may return null?

Reply:
Try: `Console.WriteLine(Activator.CreateInstance<int?>() == null);`

Reply:
Well that's awful :)

Reply:
Changed all of the object/ObjectHandle-returning overloads to be nullable.